### PR TITLE
fix broken namespecific rclink

### PR DESCRIPTION
**Pull request by alxwr**
(Puppet 3.8.6)

### CODE REVIEW

**Comment by alxwr** [comment]
The checks for 1.9.3 fail with `An error occurred while installing json_pure (2.0.2), and Bundler cannot continue.`
Succeedes for Ruby 2.0

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,6 @@ spec/fixtures
 .gitconfig
 .bundle
 Gemfile.lock
-.idea
\ No newline at end of file
+.idea
+*.swp
```
Comment (luxflux) [inline .gitignore]: These `gitignore` entries are a good fit for your global `~/.gitignore`, but please don't add them here.

Comment (alxwr) [inline .gitignore]: OK. I removed them.

**Comment by luxflux** [comment]
Thank you!

**Pull request merged**